tests/specs/core/dynamic/BeanPopulatorTest.cfc

```diff
@@ -0,0 +1,66 @@
+<cfcomponent extends="coldbox.system.testing.BaseTestCase">
```

Comment:
This looks great, however, can you move some of these tests here: https://github.com/ColdBox/coldbox-platform/blob/development/tests/specs/orm-enabled/BeanPopulatorTest.cfc

The bean populator tests need to go into the `orm-enabled` specs folder in order for them to work.